docs/vite.config.ts

```diff
@@ -6,6 +6,8 @@ import vueJsx from '@vitejs/plugin-vue-jsx'
 import i18n from '@intlify/unplugin-vue-i18n/vite'
 import autoprefixer from 'autoprefixer'
 import discardCss from 'postcss-discard-duplicates'
+import Inspect from 'vite-plugin-inspect'
+import { MarkdownTransform } from './build/plugins/markdown-transform'
```

Comment:
We should use camel case for variables normally.

Reply:
1